tests/layers/CMakeLists.txt

```diff
@@ -114,3 +68,33 @@ add_vk_layer(wrap_objects ${WRAP_SRCS})
 
 set(TEST_SRCS test.cpp vk_layer_table.cpp vk_layer_extension_utils.cpp)
 add_vk_layer(test ${TEST_SRCS})
+
+if(WIN32 OR CMAKE_GENERATOR MATCHES "^Xcode.*")
+    set(JSON_DIR "${CMAKE_CURRENT_BINARY_DIR}/$<CONFIG>")
+else()
+    set(JSON_DIR "${CMAKE_CURRENT_BINARY_DIR}")
+endif()
+
+# Run each .json.in fule through the generator
```

Comment:
This is semi-tricky.  Might want to add a comment that it is necessary to create a Cmake command file so that it can run during the build phase.